draft-ietf-quic-qlog-h3-events.md

```diff
@@ -261,6 +266,35 @@ H3StreamType =  "request" /
 ~~~
 {: #h3-streamtypeset-def title="H3StreamTypeSet definition"}
 
+## priority_updated {#h3-priorityupdated}
+Importance: Base
+
+Emitted when a stream or server push's priority is initialized or updated
+through mechanisms defined in {{!RFC9218}}. Specifically: an initial value is
+set by default, an initial value is set through an HTTP Field in a HEADERS or
+PUSH_PROMISE frame, or an updated value is set through an HTTP Field in a
+PRIORITY_UPDATE frame.
+
+Definition:
+
+~~~ cddl
+H3PriorityUpdated = {
+    ; if the prioritized element is a stream
+    ? stream_id: uint64
+
+    ; if the prioritized element is a push operation
```

Comment:
```suggestion
    ; if the prioritized element is a push stream
```

Reply:
Discussed during call: rename to `request stream` and `push stream` consistently